src/autocomplete/NotifProvider.tsx

```diff
@@ -43,7 +43,8 @@ export default class NotifProvider extends AutocompleteProvider {
         if (!this.room.currentState.mayTriggerNotifOfType('room', client.credentials.userId)) return [];
 
         const { command, range } = this.getCurrentCommand(query, selection, force);
-        if (command && command[0] && '@room'.startsWith(command[0]) && command[0].length > 1) {
+        if (command && command[0] && command[0].length > 1 &&
+            ['@room', '@channel', '@everyone', '@here'].includes(command[0])) {
```

Comment:
This changes behaviour, I'd have thought you'd want something like

```suggestion
        if (command?.[0].length > 1 &&
            ['@room', '@channel', '@everyone', '@here'].some(c.startsWith(command[0]))
        ) {
```

Reply:
Ah right